full-service/migrations/2021-03-30-021521_slip10_account_key/up.sql

```diff
@@ -1,2 +1,34 @@
-ALTER TABLE accounts
-ADD COLUMN key_derivation_version INTEGER NOT NULL DEFAULT 1;
+-- ALTER TABLE accounts ADD COLUMN key_derivation_version INTEGER NOT NULL DEFAULT 1;
```

Comment:
Chatted with @christian-oudard - this is rewriting the migration to be compatible with different SQLite3 versions